reports/JAN-13-2017.md

```diff
@@ -0,0 +1,46 @@
+# Status Report for Jan 13, 2017
+
+### Preface
+
+I wanted to try something different for this project during its development by giving a weekly summary on the progress and what is being working on.  I figured the best place is to keep the reports in the repo.  Please give your feedback on the format and let me know if this is a waste of time or if there is something better we can do.
+
+## Quick Start and Vendoring
+
+This week we merged PRs adding a quick start guide with build instructions to help get new contributors onboard quicker.  We also merged a vendoring PR so everyone has the same revision of packages for the project.  It can add a little burden on development having vendoring at this early stage, especially the way we have to do vendoring in Go, but it ensures that everyone contributing has a consistent environment.
+
+* https://github.com/docker/containerd/pull/400
+* https://github.com/docker/containerd/pull/390
+
+## Milestones
+
+I spent some time yesterday working on milestones.  The milestones are more goal focused instead of being based on a versions or a date.  We have goals that we want to hit in each of the phases and the milestones are organized that way.
+
+We are currently in the bootstrap phase.  Much of the design work, getting the project's infrastructure up (ci, vendoring, guides for development, etc) are part of this work.
```

Comment:
is there any doc or shared doc people can take a look/comment/propose changes about design of the various features?